starlingx/nfv/orchestration/doc.go

```diff
@@ -0,0 +1,13 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/* Copyright(c) 2019 Wind River Systems, Inc. */
+
+package orchestration
```

Comment:
Considering use version v1 of the nfv API. The orchestration may not be necessary.

Reply:
Done.
Updated to v1.